docs/pages/repo/docs/faq.mdx

```diff
@@ -16,6 +17,35 @@ You have two options when working with Turborepo:
 
 We recommend installing the `turbo` CLI globally. This gives you a smooth, ergonomic experience for running tasks.
 
+### Why isn't my global `turbo` working as expected?
```

Comment:
Another issue I've seen here is caused by the order that the global bin directory is referenced in your PATH. For example if yarn is included before pnpm, and you update turbo via pnpm, your version isn't going to change because turbo is going to resolve to the yarn version first.

Reply:
This is a dev-only issue but I often end up with turbo in my `.cargo/bin` as well...